sentry_sdk/serializer.py

```diff
@@ -120,7 +120,9 @@ def serialize(event, **kwargs):
     path = []  # type: List[Segment]
     meta_stack = []  # type: List[Dict[str, Any]]
 
-    keep_request_bodies = kwargs.pop("request_bodies", None) == "always"  # type: bool
+    keep_max_request_body_size = (
+        kwargs.pop("max_request_body_size", None) == "always"
+    )  # type: bool
```

Comment:
Let's keep the old variable name (`keep_request_bodies`) here, `keep_max_request_body_size` isn't very clear.

Reply:
Makes sense! Applied!